### ISSUE TITLE
Develop a brew formula

### ISSUE THREAD

**Issue by alyeffy** [bioSyntax/bioSyntax#20]
[Homebrew](https://brew.sh/) is a really useful package manager for MacOS that's used extensively. To make the installation of bioSyntax easier for vim/less/gedit, we could develop brew formula to enable installation by running just one command in the terminal.

There's [documentation](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md) for developing a formula on their GitHub. Anyone interested in working with Ruby should get involved with this!

**Comment by Ebedthan** [comment]
Hi @alyeffy, ass bioSynta can be installed from the package manager of vscode and sublime, and with the installer script provided in the bioSyntax tarball, is the idea of creating a brew formula still necessary?

**Comment by alyeffy** [comment]
Hi @Ebedthan, maybe we can let @ababaian decide if it is still an avenue we want to pursue or not. Brew is a vital package manager that practically every Mac user that has ever dipped their feet development in any way at all has installed and used it, and Mac developers will definitely use it on the regular. The main thing holding back the development of this at the time this was proposed was minimal documentation and also knowledge of the Ruby programming language required to create a brew package.

**Comment by Ebedthan** [comment]
I agree with you about brew. I also use it even on Linux. We can create the brew formula. I just want have your feedback on it's current necessity.

**Comment by ababaian** [comment]
It's for sure a nice to have feature, will let the accessibility of bioSyntax improve a bit. We should keep the issue up if someone decides to tackle it :)
